docs/API/variants.md

```diff
@@ -0,0 +1,542 @@
+- Transformation parameters: Specifies the details of the transformation(s), such as the amount to trim from each side of the dimensions to resize the image.
+
+The schema allows for the addition of variants with transformations. A variant object contains the following properties:
+
+```
+"variants": [
+    {
+        "name": "<variantName>",
+        "<transformation type>": {
+            "<key>": <value>,
+            "<key>": <value>
+        }
+    }
+]
+```
+{ data-search-exclude }
+
+Note that the `variants` key above is an optional key inside of the object of type `_media`.
+
+!!! Example
+
+    ```
+    {
+        "name": "thumbnail",
+        "trim": {
+            "top": 10,
+            "right": 10,
+            "width":200,
+            "height":200
+        }
+    }
+    ```
+    { data-search-exclude }
+
+!!! Note
+    - Media objects without any variants initially won't have the "variants" key in their object data. It will be automatically added when the first variant for that media is created.
+    - Removing all variants of an asset won't delete the "variants" key from object data.
+    - Variant names must be unique and follow the regex pattern: `^[_a-z]+$` (case insensitive).
+
+## Transformation Types
+
+Currently, the only acceptable type of transformation is `trim`.
+<!-- The transformations follow Cloudflare's transformation standards. -->
+<!-- https://developers.cloudflare.com/images/transform-images/transform-via-workers -->
+
+Trim accepts 4 properties in two possible combinations:
+
+- `top,right,bottom,left` - defines how many pixels should be trimmed from each side of the image.
+
+!!! Example
+    For variant:
+
+    ```
+    {
+        "name":"thumbnail",
+        "trim": {
+            "top":10,
+            "right":10,
+            "bottom":10,
+            "left":10
+        }
+    }
+    ```
+    { data-search-exclude }
+
+    If the original image is 400/400, it will be cropped to 380/380, cutting 10 pixels from each side.
+
+- `top,left,width,height` - defines the top left point of the crop, and number of pixels that will be cropped into the resulting image.
+
+!!! Example
+    For variant:
+
+    ```
+    {
+        "name":"thumbnail",
+        "trim": {
+            "top":10,
+            "left":10,
+            "width":50,
+            "height":50
+        }
+    }
+    ```
+    { data-search-exclude }
+
+    If the original image is 400/400, it will result in cropping a 50/50 square in the top left corner, 10 pixels away from the top/left side.
+
+!!! Note
+    Transforming works together with resizing (width/height properties used in every GET image endpoint). Note that images are transformed before they are resized.
+
+## Handling Variants
+
+Creating, removing and updating variants can be done in two ways, either by updating the media content object itself using standard endpoints for editing the content object's data or by using designated endpoints for operations on variants.
```

Comment:
Please remove mentions of designated endpoints, as they are not needed and will be removed for sure